features/step_definitions/command_steps.rb

```diff
@@ -358,6 +361,46 @@
   sle11 = "#{os_version[0, 2]}-SP#{os_version[-1]}"
   cmd = "mgr-create-bootstrap-repo -c SLE-#{os_version}-#{arch}" if os_version.include? '12'
   cmd = "mgr-create-bootstrap-repo -c SLE-#{sle11}-#{arch}" if os_version.include? '11'
-  puts 'Creating the boostrap-repo on the server: ' + cmd
+  puts 'Creating the boostrap repository on the server: ' + cmd
   $server.run(cmd, false)
 end
+
+When(/^I copy server\'s keys to the proxy$/) do
+  ['RHN-ORG-PRIVATE-SSL-KEY', 'RHN-ORG-TRUSTED-SSL-CERT', 'rhn-ca-openssl.cnf'].each do |file|
+    rc = file_extract($server, '/root/ssl-build/' + file, '/tmp/' + file)
+    raise 'File extraction failed' unless rc.zero?
+    $proxy.run('mkdir -p /root/ssl-build')
+    rc = file_inject($proxy, '/tmp/' + file, '/root/ssl-build/' + file)
+    raise 'File injection failed' unless rc.zero?
+  end
+end
+
+When(/^I configure the proxy$/) do
+  # prepare the settings file
+  settings = "RHN_PARENT=#{$server_ip}\n" \
```

Comment:
settings should be more a ruby constant (uppercase)

```ruby
SETTINGS = 'blabla'.freeze
```

Reply:
In ruby, uppercase is for constants. This is a variable.

Reply:
This a constant variable. You dont change it later after init.

Reply:
But ok